routes/index.js

```diff
@@ -3,6 +3,6 @@ const router = express.Router()
 
 router.get( '/', ( request, response ) => {
   response.render( 'index' )
-})
+}) 
```

Comment:
Trim extra whitespace at end of line (there's a setting in your editor that will do this for you automatically)